src/Tooling/Bit.Tooling.SourceGenerators/Client/BlazorSetParametersSourceGenerator.cs

```diff
@@ -14,9 +14,9 @@ public void Execute(GeneratorExecutionContext context)
             if (context.SyntaxContextReceiver is not BlazorParameterPropertySyntaxReceiver receiver)
                 return;
 
-            foreach (IGrouping<INamedTypeSymbol, IPropertySymbol> group in receiver.Properties.GroupBy(f => f.ContainingType))
+            foreach (IGrouping<ISymbol, IPropertySymbol> group in receiver.Properties.GroupBy(f => f.ContainingType, SymbolEqualityComparer.Default))
             {
-                string classSource = GeneratePartialClassToOverrideSetParameters(group.Key, group.ToList());
+                string classSource = GeneratePartialClassToOverrideSetParameters(group.Key as INamedTypeSymbol, group.ToList());
```

Comment:
Please use cast instead of as
We use as in rare cases in which there is uncertainty about the object's real type.